official/resnet/keras/keras_imagenet_benchmark.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Executes Keras benchmarks and accuracy tests."""
+from __future__ import print_function
+
+import os
+
+from absl import flags
+from absl.testing import flagsaver
+import tensorflow as tf  # pylint: disable=g-bad-import-order
+
+from official.resnet import imagenet_main
+from official.resnet.keras import keras_common
+from official.resnet.keras import keras_imagenet_main
+
+
+DATA_DIR = '/data/imagenet/'
+
+
+class KerasImagenetBenchmarkTests(object):
+  """Benchmarks and accuracy tests for KerasCifar10."""
+
+  local_flags = None
+
+  def __init__(self, output_dir=None):
+    self.oss_report_object = None
+    self.output_dir = output_dir
+
+  def keras_resnet50_short_1_gpu(self):
+    """Test Keras model with Keras fit/dist_strat quick test."""
+    self._setup()
+    flags.FLAGS.num_gpus = 1
+    flags.FLAGS.data_dir = DATA_DIR
+    flags.FLAGS.batch_size = 32
```

Comment:
For the short benchmark, do we want to run at 128 batch size (must also set `skip_eval` to `True` for now to avoid OOM)?

Reply:
Hm, we cannot set skip_eval to True for accuracy tests (which is what these are, as of now). Perhaps we will need separate performance and convergence tests so we can use different batch size there?

Reply:
This test is for "testing" and will not be run for real.  I also want it to run on a GTX-1080 so I need a smaller batch-size.  i will look at renaming it in the next round.